container/incremental_load.sh.tpl

```diff
@@ -277,6 +277,18 @@ if [[ "%{run}" == "True" ]]; then
   # We can clean up before we exec docker, since the exit handler will no longer run.
   cleanup
 
+  # Bash treats empty arrays as unset variables for the purposes of `set -u` - work around this.
+  args=(%{run_statement})
+  if [[ ${#docker_args[@]} -gt 0 ]]; then
+    args+=("${docker_args[@]}")
+  fi
+  args+=("%{run_tag}")
+  if [[ ${#container_args[@]} -gt 0 ]]; then
+    args+=("${container_args[@]}")
+  else
+    args+=("-")
+  fi
```

Comment:
Please use the original form of `"${container_args[@]:-}"` or remove the `else` part. As if it's not set, it is empty but not `-`:
For more details, please see https://tldp.org/LDP/abs/html/parameter-substitution.html